Router definition in glossary (#32231)
mdn/content: files/en-us/glossary/routers/index.md

There are three definitions for **routers** on the web:

> 1. For the network layer, the router is a networking device that decides data {{Glossary('Packet')}}s directions.

Reviewer: Minor one for you to consider:

```suggestion
1. For the network layer, the router is a networking device that decides where to direct data {{Glossary('Packet', 'Packets')}}.
```

Reviewer 2: I was going to make a similar suggestion, except why capitalize "packets"?

> For the network layer, the router is a networking device that decides where to direct {{Glossary('Packet', 'data packets')}}.

Author: Arggg
```suggestion
1. For the network layer, the router is a networking device that decides where to direct {{Glossary('Packet', 'data packets')}}.
```

There are three definitions for **routers** on the web:

> 1. In computer networing, a router is a device that forwards {{Glossary("Packet", "data packets")}} between computer networks.

Author: ```suggestion
1. In computer networking, a router is a device that forwards {{Glossary("Packet", "data packets")}} between computer networks.
```

> 5. For a {{Glossary('SPA', 'Single-page application')}}, a router is a library that decides what web page is presented by a given {{Glossary('URL')}}.
>    This middleware module is used for all URL functions, as these are given a path to a file that is rendered to open the next page.
> 7. In the implementation of an {{Glossary("API", "Application Programming Interface (API))}}, a router is a software component that parses a request and directs or routes the request to various handlers within a program.

Author: ```suggestion
2. For a {{Glossary('SPA', 'Single-page application')}}, a router is a library that decides what web page is presented by a given {{Glossary('URL')}}.
   This middleware module is used for all URL functions, as these are given a path to a file that is rendered to open the next page.
3. In the implementation of an {{Glossary("API", "Application Programming Interface (API))}}, a router is a software component that parses a request and directs or routes the request to various handlers within a program.
```
